en/peripherals/camera.md

```diff
@@ -33,17 +33,20 @@ Mode | Description
 
 ## Trigger Hardware Configuration {#hardware_setup}
 
-You can choose which pins to use for triggering using the [TRIG_PINS](../advanced_config/parameter_reference.md#TRIG_PINS) parameter.
+On Pixhawk FMUv5x boards use the board-specific camera capture pin (PI0) for triggering
+(this is automatically enabled when `TRIG_INTERFACE` is 0/GPIO or 1:Seagull).
+
+For other FMUv versions you choose the pins to use for GPIO or Seagull-based triggering using the [TRIG_PINS](../advanced_config/parameter_reference.md#TRIG_PINS) parameter.
```

Comment:
Same question really - it isn't clear from the docs how the PWM option works - what pin is used for the output, and how you tell PW4 what PWM value must be used to trigger.

Reply:
`TRIG_PINS` defines the output(s): e.g. 56 you will have pwm signal on pins 5+6 (duplicated). Seagull requires 2 signals, so you have to use 2 pins (lower pin number is the channel 2 (mode), higher pin number is the channel 1 (trigger)).
```suggestion
For other FMUv versions you choose the pins to use for GPIO, PWM or Seagull-based triggering using the [TRIG_PINS](../advanced_config/parameter_reference.md#TRIG_PINS) parameter.
```
Is that clear?